docs/_user_guide/best_practices_matlab_slurm_job.md

```diff
@@ -1,33 +1,33 @@
 ## Matlab Example Slurm Script
 The simplest Matlab script looks like:
 
-```matlab
+```matlab title="Matlab Code"
 disp('Hello world')
 ```
 
 Save this line to a new file called `hello_world.m`. 
 
-## Available Matlab versions on Yen10
+## Available Matlab versions on `Yen10`
 Several version of Matlab are installed on the yens and the default version is designated with `(D)`:
 
-```bash
+```bash title="Terminal Command"
 module avail matlab
 ```
 
 Load the version you want:
 
-```bash
+```bash title="Terminal Command"
 module load matlab
 ```
 
 If you do not specify the version, `matlab/R2019b` will be loaded by default.
```

Comment:
The default is different now. Change this line to say "If you do not specify the version (`module load matlab`), the version marked with a `(D)` will be loaded by default."